code/modules/mob/dead/observer/orbit.dm

```diff
@@ -79,6 +80,9 @@
 				alive += list(serialized)
 
 				var/datum/mind/mind = M.mind
+				if(mind.special_role in list(SPECIAL_ROLE_ERT, SPECIAL_ROLE_DEATHSQUAD, SPECIAL_ROLE_SYNDICATE_DEATHSQUAD))
```

Comment:
will this instantiate this list every time? if so you may want to pull it out of the for loop, or out of the proc altogether, though that might be overkill

Reply:
discussed offline, probably overkill